src/styles/animation.scss

```
@@ -0,0 +1,46 @@
+.item-enter {
+  max-height: 0;
+}
+
+.item-enter-active {
+  overflow: hidden;
+  max-height: 58px;
+  transition: max-height 0.3s ease-in-out;
+}
```

Please follow to the BEM structure 
```suggestion
.item-enter {
  max-height: 0;
  
  &-active {
  overflow: hidden;
  max-height: 58px;
  transition: max-height 0.3s ease-in-out;
 }
}
```